src/test/java/io/stargate/sgv2/jsonapi/api/v1/InsertInCollectionIntegrationTest.java

```diff
@@ -1005,138 +1004,6 @@ void successForPathOverlapping() {
     }
   }
 
-  @Nested
```

Comment:
Refactoring: moving to new test class ("InsertLexicalInCollectionIntegrationTest" below)